### DISCUSSION TITLE
curses.h

### DISCUSSION THREAD

**Discussion by pabenito** [cmus/cmus#1062] [General]
make
   CC     ape.o
   CC     browser.o
   CC     buffer.o
   CC     cache.o
   CC     channelmap.o
   CC     cmdline.o
   CC     cmus.o
   GEN    .version
   CC     command_mode.o
In file included from command_mode.c:19:
command_mode.h:27:10: fatal error: curses.h: No such file or directory
   27 | #include <curses.h>
      |          ^~~~~~~~~~
compilation terminated.

**Comment by pgaskin** [comment]
Which distro are you using?

If you are using a Debian/Ubuntu derivative, see here for the dependencies: https://github.com/cmus/cmus/blob/a9ac3dec00a4c1a8761d6684509e51157300dc63/.github/workflows/build.yml#L30-L35
